src/winetricks

```diff
@@ -9424,7 +9424,10 @@ load_msxml3()
     rm "$W_SYSTEM32_DLLS"/msxml3.dll
     w_override_dlls native msxml3
     w_try_cd "$W_CACHE/$W_PACKAGE"
-    w_try "$WINE" msiexec /i msxml3.msi $W_UNATTENDED_SLASH_Q
+    #Workaround wine bug by forcing silent install
+    #See: https://github.com/Winetricks/winetricks/issues/1086
+    #FIXME: report bug upstream or link to already reported bug
```

Comment:
Please do that before I merge it ;).

Zeb implied it may be https://bugs.winehq.org/show_bug.cgi?id=26925, but I'm not 100% sure that's accurate. @zfigura do you know if this is 26925 or a separate issue?

Reply:
I believe I remember testing it and being confident it was 26925. The only thing that confused me was why it only showed up when it did; perhaps there's some odd effect from custom action separation.

Reply:
Okay, great, thanks! Let's use that bug then @gverm (behind w_workaround_wine_bug) so users get notified of what's going on.